blueprints/grafana/config.yml

```
@@ -0,0 +1,5 @@
+blueprint:
+  grafana:
+    pkgs: grafana5
+    vars: link_influxdb database
```

Vars can't handle sub vars. They need the full variable.
```suggestion
    vars: link_influxdb database_name database_user database_database database_password
```